### PR TITLE
Add new model of TS0121 Tuya plug

### PR DESCRIPTION
**Pull request by Adminiuga**
Add a new model of TS0121 Tuya plug. 
Fixes https://github.com/zigpy/zha-device-handlers/issues/1299 
I don't have such a plug so this needs testing to confirm it is working correctly.

### CODE REVIEW

**Comment by ali1234** [comment]
This certainly causes the quirk to be applied to my devices. The custom clusters are now shown in the cluster manager. The cluster manager works now and I can get the value of current_divisor, and it is 1000 as expected when the quirk is activated.

Tomorrow I will verify that the readings really are correct using my 2kW fan heater.

File: zhaquirks/tuya/ts0121_plug.py [hunk 1]
```diff
@@ -62,3 +64,65 @@ class Plug(CustomDevice):
...
+
+class TS0121B(CustomDevice):
+    """Tuya TS0121 plug with restore tuya power state support and ZGP endpoint."""
+
+    signature = {
+        MODEL: "TS0121",
+        ENDPOINTS: {
+            # <SimpleDescriptor endpoint=1 profile=260 device_type=81
+            # device_version=1
+            # input_clusters=[0, 4, 5, 6, 9, 1794, 2820]
```
Comment (javicalle) [inline zhaquirks/tuya/ts0121_plug.py:77]: The device signature is wrong.
Maybe copy&paste?

**Review by javicalle — CHANGES_REQUESTED** [review]

File: zhaquirks/tuya/ts0121_plug.py (hunk 1 above)
Comment (ali1234) [reply zhaquirks/tuya/ts0121_plug.py:77]: `input_clusters` comment is wrong for both devices too. Neither defines 0x0009 "Alarms" and the new one is obviously completely different...

**Comment by ali1234** [comment]
After running heater for 6 minutes I have this:

image

These readings seem to be correct.

File: zhaquirks/tuya/ts0121_plug.py [hunk 2]
```diff
@@ -62,3 +64,65 @@ class Plug(CustomDevice):
...
+            # output_clusters=[10, 25]>
```
Comment (Adminiuga) [inline zhaquirks/tuya/ts0121_plug.py:78]:
```suggestion
```

File: zhaquirks/tuya/ts0121_plug.py (hunk 1 above)
Comment (Adminiuga) [reply zhaquirks/tuya/ts0121_plug.py]: fixed

**Comment by javicalle** [comment]
LGTM

**Pull request merged**